templates/action-group.json

```diff
@@ -8,14 +8,26 @@
     "alertShortName": {
       "type": "string"
     },
-    "alertEmailReceivers": {
-      "type": "array"
+    "emailReceiverName": {
+      "type": "string",
+      "defaultValue": "emailReceiver",
+      "metadata": {
+        "description": "Name of the email receiver"
+      }
+    },
+    "emailReceiverAddress": {
+      "type": "string",
+      "metadata": {
+        "description": "Email address of the receiver"
+      }
     },
-    "alertSmsReceivers": {
-      "type": "array"
+    "location": {
+      "type": "string",
+      "defaultValue": "global"
     },
-    "alertWebhookReceivers": {
-      "type": "array"
+    "env": {
+      "type": "string",
+      "defaultValue": "sandbox"
```

Comment:
any reason to have a default for this?

Reply:
not particularly. I was thinking it would serve as a buffer in case it was missed - but probably the failure would be more appropriate in that case anyway

Reply:
please remove